schemas/rules/definitions.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$id": "https://eq.ons.gov.uk/rules/definitions.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "operators": {
+    "description": "All operators",
+    "oneOf": [
+      { "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/equal.json" },
+      {
+        "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/not-equal.json"
+      },
+      {
+        "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/less-than.json"
+      },
+      {
+        "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/less-than-or-equal.json"
+      },
+      {
+        "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/more-than.json"
+      },
+      {
+        "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/more-than-or-equal.json"
+      },
+      { "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/in.json" },
+      { "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/any.json" },
+      { "$ref": "https://eq.ons.gov.uk/rules/operators/comparison/all.json" },
+      { "$ref": "https://eq.ons.gov.uk/rules/operators/date-offset.json" },
+      { "$ref": "https://eq.ons.gov.uk/rules/operators/boolean/and.json" },
+      { "$ref": "https://eq.ons.gov.uk/rules/operators/boolean/or.json" },
+      { "$ref": "https://eq.ons.gov.uk/rules/operators/boolean/not.json" }
+    ]
+  },
+  "any_value": {
+    "description": "Any value",
+    "oneOf": [
+      { "type": ["number", "string", "array", "null"] },
```

Comment:
Shouldn't our `non-empty` string be used here?

Reply:
I don't think so, the rules will work with empty strings.

Reply:
I was wondering when that would be valid?. For unanswered it would be `null`, so we wouldn't ever compare empty strings?

Reply:
My thinking is that is not something that would be known until runtime of an implementation, and the rules are not tied to our implementation. I also think it makes sense to make it as permissive as possible for ease of extending it in the future.